mage/query-modules/cpp/community_detection.md

```diff
@@ -23,39 +23,39 @@ export const Highlight = ({children, color}) => (
 
 ## Abstract
 
-This query module makes available the [Louvain method](https://en.wikipedia.org/wiki/Louvain_method) <sup>[1]</sup> for community
+This query module makes available the [Louvain method](https://en.wikipedia.org/wiki/Louvain_method)<sup>[1]</sup> for community
```

Comment:
Shouldn't it be
This query module makes the Louvain method available for community detection, ... ?

Reply:
Agree with that, but your wording risks misinterpretation as "This query module makes the Louvain method {available for community detection} [...]". Does "This query module provides the Louvain method [...]" work?

Reply:
Thanks for the suggestions, just committed the former!